third_party/plaidml/workspace.bzl

```diff
@@ -0,0 +1,20 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def clean_dep(dep):
+    return str(Label(dep))
+
+def repo():
+    http_archive(
+        name = "com_intel_plaidml",
+        build_file = clean_dep("//third_party/plaidml:BUILD.bazel"),
```

Comment:
Drop this, we shouldn't need to provide a custom `BUILD` file for this repository because it's already a native bazel repository.